docusaurus/docs/recipes/video-player.md

```diff
@@ -0,0 +1,99 @@
+---
+id: video-player
+title: Adding a video player
+sidebar_label: Video player
+---
+
+It's often normal to have the necessity of having videos in your website. In this recipe we will show you how to use [`react-player`](https://github.com/CookPete/react-player) and some common use cases.
+
+## Walk-through
+
+### 1. Install `react-player`
+
+```sh
+npm i react-player
+```
+
+### 2. Use `ReactPlayer` in your project
+
+You just need to import `ReactPlayer` and start using it.
+
+```js
+// ...
+import ReactPlayer from 'react-player';
+
+const Home = () => (
+    <div className={ styles.home }>
+        <ReactPlayer url="https://www.video-url.com" />
+    </div>
+);
+```
+
+### 3. Usage
+
+Now let's go through some common use cases that might be helpful.
+
+> ⚠️ By default `react-player` sets a width of 640px and a height of 360px to the player. You should change it in order to fit your needs.
+
+#### 1. Autoplay
+
+To enable autoplay you need to use `playing` prop.
+
+```js
+<ReactPlayer url="https://www.video-url.com" playing />
+```
+
+Be sure to check the different browser policies for autoplaying videos as they are different and you might need additional configuration. For example on Chrome you might need to mute the video in order to autoplay it.
+
+#### 2. Enable/disable default controls
+
+To enable or disable the video controls you can use the `controls` prop.
+
+```js
+<ReactPlayer url="https://www.video-url.com" controls />
+```
+
+#### 3. Disable download
+
+The download of videos is enable by default. To disable it you need to change the [`controlsList`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/controlsList) using the `config` prop.
```

Comment:
Typo, should be `enabled`.

Also, i would write it `need do change the controlList property, using the...`, i.e. adding property after mentioning the property. This might seem like nickpicking but it incentivizes maintaining terminology across documentations, which keeps things clearer and traceable for the reader.